lib/rust/ensogl/examples/dom-symbols/src/lib.rs

```diff
@@ -83,17 +83,18 @@ pub fn main() {
     let camera = scene.camera();
     let navigator = Navigator::new(scene, &camera);
     let sprite_system = SpriteSystem::new("test_sprite_system", alignment::Dim2::center());
-    world.add_child(&sprite_system);
+    world.add_child(&sprite_system.symbol);
 
     let dom_front_layer = &scene.dom.layers.front;
     let dom_back_layer = &scene.dom.layers.back;
 
-    let mut sprites: Vec<Sprite> = default();
+    let mut sprites: Vec<SpriteObject> = default();
```

Comment:
Can we rename `SpiteObject` to `Sprite` and `Sprite` to something else? Sprites by definitions are things displayed on the screen, so I thing the name of just "Sprite" will be better for the user.